tests/test_errors.py

```diff
@@ -1074,3 +1075,11 @@ def test_hide_input_in_json() -> None:
 
     for error in exc_info.value.errors(include_input=False):
         assert 'input' not in error
+
+
+def test_validation_error_pickle() -> None:
+    s = SchemaValidator({'type': 'int'})
+    with pytest.raises(ValidationError) as exc_info:
+        s.validate_python('definitely not an int')
+
+    pickle.loads(pickle.dumps(exc_info.value))
```

Comment:
Should this test that the unpickled value is equivalent to the original?

Reply:
I'll check `.errors` are equivalent, as I want to steer clear from implementing equality for the whole objects (as that's a much bigger change).